### PR TITLE
Paa s suitability matrix

### PR DESCRIPTION
**Pull request by AshiqaliDamani**

### CODE REVIEW

**Comment by AshiqaliDamani** [comment]
Initial addition of Gov UK PaaS vs Azure CIP Suitability Matrix

File: source/documentation/guides/govuk-paas-suitability.html.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+| Auto and horizontal scaling    | Yes, configurable       | Yes, configurable      |
+| Rapid deployment    | Yes       | Yes      |
+| Nature of Ecosystem    | IaaS and PaaS       | PaaS only      |
+| Compliance with DfE Standards    | Yes       | Yes      |
+| Public facing type apps/services    | Configurable       | Suited (configurational setup)     |
+| Lightweight Apps    | Acceptable       | Suited (Time to market)     |
+| Time-Bound Prototypying    | Acceptable       | Suited (Time to market)      |
+| OTS Platform Security    | Good, via security rules       | Good, built in the platform      |
+| Areas of Operation    | Full techn stack       | PaaS only      |
+| GitHub Integration    | TBD       | Configured by service      |
```
Comment (saliceti) [inline source/documentation/guides/govuk-paas-suitability.html.md:15]: Github is independent from the hosting platform.
Azure devops integrates well with Github. I think both entries could be "Configured by service"

File: source/documentation/guides/govuk-paas-suitability.html.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+| Disaster Recovery and Load Balancing    | Yes, configurable       | Yes, but not configurable at a low level      |
+| Auto and horizontal scaling    | Yes, configurable       | Yes, configurable      |
+| Rapid deployment    | Yes       | Yes      |
+| Nature of Ecosystem    | IaaS and PaaS       | PaaS only      |
+| Compliance with DfE Standards    | Yes       | Yes      |
+| Public facing type apps/services    | Configurable       | Suited (configurational setup)     |
+| Lightweight Apps    | Acceptable       | Suited (Time to market)     |
+| Time-Bound Prototypying    | Acceptable       | Suited (Time to market)      |
+| OTS Platform Security    | Good, via security rules       | Good, built in the platform      |
+| Areas of Operation    | Full techn stack       | PaaS only      |
```
Comment (saliceti) [inline source/documentation/guides/govuk-paas-suitability.html.md:14]: Typo

File: source/documentation/guides/govuk-paas-suitability.html.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+| GitHub Integration    | TBD       | Configured by service      |
+| Platform design lends itself to loosley coupled applications | Nothing preventing it       | Yes      |
+| Containerised App Deployment  | Configurable       | Suited      |
+| Multi-region    | Yes, Locations TBD       | Yes, IRE and London      |
```
Comment (saliceti) [inline source/documentation/guides/govuk-paas-suitability.html.md:18]: Azure (CIP) is only West-Europe (Netherlands) so it's not multi region.

File: source/documentation/guides/govuk-paas-suitability.html.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+| RBAC Governance    | Excellent       | Poor      |
+| Enforced User Security Policies    | Yes       | No      |
+| Enforced Platform Security    | Yes       | Yes      |
+| Aligned to Twelve Factor Principles    | Not specfically       | Yes      |
+| Provision and Remove Environments    | Laboured       | Fast      |
+| Commercial Support    | Microsoft, Bytes       | GDS      |
+| Community Support    | CIPS, Azure community       | GDS, Cloudfoundry, Cross-Government       |
+| Charging Granularity    | Minute (Azure)       | Hour (via Marketplace)      |
+| Time to Provision    | Days / weeks       | Hours / days      |
+| Current Utilisation No of Applications/services and %    | ~44, >95%       | ~4, <5%      |
```
Comment (saliceti) [inline source/documentation/guides/govuk-paas-suitability.html.md:39]: There are 10 production applications on paas now, and a number in development. We could up that number?

File: source/documentation/guides/govuk-paas-suitability.html.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+| Event Auditing    | Very Strong       | Strong      |
+| Smaller sized applications/services for iterative development    | Configurable       | Suited      |
+| Monitoring Solution   | Yes       | Limited via Portal (Additonal tools are pluggable)       |
+| Deployed by buildpacks/docker    | Yes       | Yes      |
+| Accessuble via HTTP or TCP    | TBD       | Yes      |
```
Comment (saliceti) [inline source/documentation/guides/govuk-paas-suitability.html.md:44]: Typo

File: source/documentation/guides/govuk-paas-suitability.html.md (hunk 1 above)
Comment (AshiqaliDamani) [reply source/documentation/guides/govuk-paas-suitability.html.md]: Corrected 'Prototyping' and 'technical'

Comment (AshiqaliDamani) [reply source/documentation/guides/govuk-paas-suitability.html.md]: Updated the GitHub entry

File: source/documentation/guides/govuk-paas-suitability.html.md (hunk 5 above)
Comment (AshiqaliDamani) [reply source/documentation/guides/govuk-paas-suitability.html.md]: Corrected typos

File: source/documentation/guides/govuk-paas-suitability.html.md (hunk 4 above)
Comment (AshiqaliDamani) [reply source/documentation/guides/govuk-paas-suitability.html.md]: Updated the Application tally and percentages

File: source/documentation/guides/govuk-paas-suitability.html.md (hunk 2 above)
Comment (AshiqaliDamani) [reply source/documentation/guides/govuk-paas-suitability.html.md]: Corrected

**Review by AshiqaliDamani — COMMENTED** [review]
Made further updates and corrections

**Review by saliceti — APPROVED** [review]

**Pull request merged**